distrobox-host-exec

```diff
@@ -75,6 +79,10 @@ if [ -z "${host_command}" ]; then
 				printf "distrobox: %s\n" "${version}"
 				exit 0
 				;;
+			-nc | --no-confirm) 
```

Comment:
I'm ok with the idea, can you use `-Y/--yes` instead like in other parts of distrobox? check `distrobox-rm` for a reference
also using `0` and `1` instead of yes and no :+1: